Analysis/Tasks/PWGHF/taskBPlus.cxx

```diff
@@ -45,38 +63,36 @@ struct TaskBplus {
   Configurable<double> cutEtaCandMax{"cutEtaCandMax", -1., "max. cand. pseudorapidity"};
 
   Filter filterSelectCandidates = (aod::hf_selcandidate_d0::isSelD0 >= d_selectionFlagD0 || aod::hf_selcandidate_d0::isSelD0bar >= d_selectionFlagD0bar);
+  Filter positiveTracks = aod::track::signed1Pt >= 0.f;
```

Comment:
hi @aalkin I thought a bit. I would probably avoid doing the charge selection upfront cause we will have to add in the code the possibility of doing the B-.

Reply:
I can fix it myself in any case no worry :)

Reply:
Then it would make sense to use partitions, you can add one for negative tracks later. I'll add a commit later today.